### ISSUE TITLE
Plugins/Editor/VSCode in assets but no new tab in Preferences

### ISSUE THREAD

**Issue by Haub100** [dotBunny/VSCode#65]
This is a strange issue. One of my unity projects doesn't seem to accept VSCode integration. This worked fine on all of my other projects and originally worked on this one but now integration has disappeared. I don't know what I did to cause this change. Even deleting that folder and recopying that file does not work. Any ideas or at least a way I completely delete VSCode integration to reinstall?

I'm on Windows 10 x64 bit with unity 5.3.2f1

**Comment by fsommar** [comment]
The same thing happened to me. When I looked in the console, it appeared as if the mono runtime had crashed while trying to compile to project. In other words, make sure that you don't have any errors in the console. This presumably prevented the runtime from continuing and eventually loading this plugin.

In my case it was caused by something involving a generic struct (and possibly making it nullable?). It is worth a shot at least to check the console!

See http://answers.unity3d.com/questions/497360/internal-compiler-error-with-no-output.html for reference.
